internal/client.go

```diff
@@ -418,9 +417,9 @@ type (
 		WorkflowTaskTimeout time.Duration
 
 		// WorkflowIDReusePolicy - Whether server allow reuse of workflow ID, can be useful
-		// for dedup logic if set to WorkflowIdReusePolicyRejectDuplicate.
-		// Optional: defaulted to WorkflowIDReusePolicyAllowDuplicateFailedOnly.
-		WorkflowIDReusePolicy WorkflowIDReusePolicy
+		// for dedup logic if set to RejectDuplicate.
+		// Optional: defaulted to AllowDuplicateFailedOnly.
```

Comment:
I believe default is AllowDuplicate now.

Reply:
:facepalm: